### PR TITLE
Working PathPlanner Paths

### PR DESCRIPTION
**Pull request by mxweII**
Got the paths working

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -49,7 +49,31 @@ def includeDesktopSupport = false
...
+
+// task(checkAkitInstall, dependsOn: "classes", type: JavaExec) {
+//     mainClass = "org.littletonrobotics.junction.CheckInstall"
+//     classpath = sourceSets.main.runtimeClasspath
+// }
+// compileJava.finalizedBy checkAkitInstall
 dependencies {
+    // ...
+    // def akitJson = new groovy.json.JsonSlurper().parseText(new File(projectDir.getAbsolutePath() + "/vendordeps/AdvantageKit.json").text)
+    // annotationProcessor "org.littletonrobotics.akit.junction:junction-autolog:$akitJson.version"
```
Comment (notrandomness) [inline build.gradle]: Let's go ahead and remove these commented out lines.

File: src/main/java/frc/robot/SwerveModule.java [hunk 2]
```diff
@@ -89,4 +92,12 @@ public SwerveModulePosition getPosition(){
             Rotation2d.fromRotations(mAngleMotor.getPosition().getValue())
         );
     }
-}
\ No newline at end of file
+    public SwerveModuleState runSetpoint(SwerveModuleState state){
+        var optomizedState = SwerveModuleState.optimize(state, Rotation2d.fromRotations(mAngleMotor.getPosition().getValue()));
```
Comment (notrandomness) [inline src/main/java/frc/robot/SwerveModule.java]: Optimize is spelled wrong

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 3]
```diff
@@ -16,22 +16,26 @@
...
 
-public class Swerve extends SubsystemBase {
+public class Swerve extends SubsystemBase{
     public SwerveDriveOdometry swerveOdometry;
     public SwerveModule[] mSwerveMods;
     public Pigeon2 gyro;
     private SwerveDrivePoseEstimator swerveEstimator;
     private static Swerve swerve;
-
+    private final SwerveRequest.ApplyChassisSpeeds autoRequest = new SwerveRequest.ApplyChassisSpeeds();
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: Where are we using this?

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 4]
```diff
@@ -174,11 +181,20 @@ public ChassisSpeeds getRobotRelativeSpeeds(){
...
     public void driveRobotRelative(ChassisSpeeds robotRelativeSpeeds){
-        var states = Constants.Swerve.swerveKinematics.toSwerveModuleStates(robotRelativeSpeeds);
-    
-        SwerveDriveKinematics.desaturateWheelSpeeds(states, Constants.Swerve.maxSpeed);
-    
-        setModuleStates(states);
+        //robotRelativeSpeeds.omegaRadiansPerSecond = 0.0;
+        SmartDashboard.putNumber("omega radians per second", robotRelativeSpeeds.omegaRadiansPerSecond);
+        SmartDashboard.putNumber("x speed", robotRelativeSpeeds.vxMetersPerSecond);
+        SmartDashboard.putNumber("y speed", robotRelativeSpeeds.vyMetersPerSecond);
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: Let's go ahead and comment out these dashboard outputs.

**Review by notrandomness — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (mxweII) [reply build.gradle]: done

File: src/main/java/frc/robot/subsystems/Swerve.java (hunk 3 above)
Comment (mxweII) [reply src/main/java/frc/robot/subsystems/Swerve.java]: It was leftover from advantage kit, removed

File: src/main/java/frc/robot/subsystems/Swerve.java (hunk 4 above)
Comment (mxweII) [reply src/main/java/frc/robot/subsystems/Swerve.java]: done

File: src/main/java/frc/robot/SwerveModule.java (hunk 2 above)
Comment (mxweII) [reply src/main/java/frc/robot/SwerveModule.java]: fixed

**Review by notrandomness — APPROVED** [review]

**Review by T-Wezzy — APPROVED** [review]
looks good 👍

**Pull request merged**